templates/blog/macros.html

```diff
@@ -19,7 +19,7 @@
     <a href={{ url }} lang={{ article.language }}>
       <h1>{{ article.title|title }}</h1>
       <figure>
-        <img src="{{ article.image | default('/static/uploads/articles/default.png', true) }}" width=100%>
+        <img src="{{ article.image | default('../static/images/article.png', true) }}" width=100%>
```

Comment:
`../static/` does not make sense. `/static/` is at the root url already.
However, `url_for('static', filename='images/article.png')` would take all its sense ;)